source/mainnet/net/references/grpc.rst

```diff
@@ -0,0 +1,886 @@
+
+      .. literalinclude:: grpc-json-schemas/InvokeContract.json
+         :language: json
+
+Baking
+======
+
+.. function:: GetPoolStatus(blockHash, passiveDelegation, bakerId) -> ?PoolStatus
+
+   Get the status of a pool.
+   If ``passiveDelegation == true``, this returns the status for the passive delegators.
+   Otherwise, it returns the status for the baker with the specified ID (if it exists).
+
+   :param blockHash: The given block.
+   :type blockHash: |grpc-block-hash|_
+   :param Bool passiveDelegation: Whether the request is for passive delegation or a
+                             specific baker.
+   :param Integer bakerId: The baker id to get the status of.
+   :returns: The status of the pool.
+   :rtype: ``?PoolStatus`` (see JSON schema below)
+
+   .. collapse:: View JSON schema
+
+      .. literalinclude:: grpc-json-schemas/GetPoolStatus.json
+         :language: json
+
+
+.. function:: GetRewardStatus(blockHash) -> ?RewardStatus
+
+   Get an overview of the balance of special accounts in the given block.
+
+   :param blockHash: The given block.
+   :type blockHash: |grpc-block-hash|_
+   :returns: The reward status in the given block.
+   :rtype: ``RewardStatus`` (see JSON schema below)
+
+   .. collapse:: View JSON schema
+
+      .. literalinclude:: grpc-json-schemas/GetRewardStatus.json
+         :language: json
+
+.. function:: GetBirkParameters(blockHash) -> ?BirkParameters
+
+   Get an overview of the parameters used for baking.
+
+   :param blockHash: The given block.
+   :type blockHash: |grpc-block-hash|_
+   :returns: The parameters used for baking in the given block.
+   :rtype: ``?BirkParameters`` (see JSON schema below)
+
+   .. collapse:: View JSON schema
+
+      .. literalinclude:: grpc-json-schemas/GetBirkParameters.json
+         :language: json
+
+.. function:: GetBakerList(blockHash) -> ?[BakerId]
+
+   Get a list of all baker IDs registered at that block in ascending order. Or
+   ``null``, if the block is invalid.
+
+   :param blockHash: The given block.
+   :type blockHash: |grpc-block-hash|_
+   :returns: A list of baker IDs.
+   :rtype: ``?[BakerId]`` (see JSON schema below)
+
+   .. collapse:: View JSON schema
+
+      .. literalinclude:: grpc-json-schemas/GetBakerList.json
+         :language: json
+
+.. function:: StartBaker() -> Bool
+
+   Start the baker.
+
+   :returns: Whether starting the baker succeeded.
+   :rtype: Bool
+
+.. function:: StopBaker() -> Bool
+
+   Stop the baker.
+
+   :returns: Whether stopping the baker succeeded.
+   :rtype: Bool
+
+The node
+========
+
+.. function:: NodeInfo() -> NodeInfoResponse
+
+   Get information about the running node.
+
+   :returns: Information about the running node.
+   :rtype: |NodeInfoResponse|_
+
+.. function:: PeerVersion() -> String
+
+   Get the version of the node software.
+
+   :returns: The version of the node software.
+   :rtype: String
+
+.. function:: PeerUptime() -> UInt64
+
+   Get the uptime of the *node* in milliseconds.
```

Comment:
I'm not sure what the reason for putting node in italics is here, and in the 2 calls below.

Reply:
I think the name `PeerUptime` is a bit misleading, so I wanted to clarify that it refers to the node itself as opposed to a peer.